File: src/main/java/ludwig/model/PlaceholderNode.java
package ludwig.model;

public class PlaceholderNode extends Node<PlaceholderNode> {
    public String parameter() {
        return parameter;
    }

    public PlaceholderNode parameter(String parameter) {
        this.parameter = parameter;
        return this;
    }

    private String parameter;



    @Override
    public <T> T accept(NodeVisitor<T> visitor) {
        return visitor.visitPlaceholder(this);
    }

    @Override
    public String toString() {
        return "<" + parameter + ">";
    }
}

File: src/main/kotlin/ludwig/model/PlaceholderNode.kt
package ludwig.model

class PlaceholderNode : Node() {
    var parameter: String? = null


    override fun <T> accept(visitor: NodeVisitor<T>): T {
        return visitor.visitPlaceholder(this)
    }

    override fun toString(): String {
        return "<$parameter>"
    }
}
